### PR TITLE
The source of configuration properties is recorded in a test artifact.

### PR DESCRIPTION
**Pull request by techcobweb**
Signed-off-by: Mike Cobbett <77053+techcobweb@users.noreply.github.com>

# Why ?
See [Save the overrides for a test run and make accessible #1919](https://github.com/galasa-dev/projectmanagement/issues/1919)

### CODE REVIEW

**Comment by galasa-team** [comment]
Build failed, see http://localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/#/namespaces/galasa-build/pipelineruns/repo-framework-pr-637-tmjg8 for details. If you are unable to do so, please contact a member of the Galasa team.

**Review by eamansour — APPROVED** [review]

File: galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/internal/runner/BundleManager.java [hunk 1]
```diff
@@ -28,7 +33,9 @@ public void loadAllGherkinManagerBundles(RepositoryAdmin repositoryAdmin, Bundle
     @Override
     public void loadBundle(RepositoryAdmin repositoryAdmin, BundleContext bundleContext, String bundleSymbolicName)
             throws FrameworkException {
-        BundleManagement.loadBundle(repositoryAdmin, bundleContext, bundleSymbolicName);
+        try {
+            BundleManagement.loadBundle(repositoryAdmin, bundleContext, bundleSymbolicName);
+        }
```
Comment (eamansour) [inline galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/internal/runner/BundleManager.java:38]: Looks like this is missing a catch

Comment (techcobweb) [reply galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/internal/runner/BundleManager.java:38]: it wasn't intentional. OOps.

**Comment by galasa-team** [comment]
Build successful

**Pull request merged**